src/search.cpp

```diff
@@ -958,7 +962,8 @@ namespace {
               }
           }
           else if (   depth < 3 * ONE_PLY
-                   && (     mp.see_sign() < 0
+                   && (  see < VALUE_ZERO     
+					   ||  mp.see_sign() < 0
                        || (!mp.see_sign() && pos.see_sign(move) < VALUE_ZERO)))
               continue;
       }
```

Comment:
You are checking for see < VALUE_ZERO but mp.see_sign() need not be zero ... not sure if the check is proper here ...